redact/__init__.py

```diff
@@ -2,13 +2,13 @@
 Python client for "brighter Redact"
 """
 
-__version__ = "6.0.0"
+__version__ = "7.0.0"
 
 from .errors import RedactConnectError, RedactResponseError
-from .v3.data_models import (
+from .v3.data_models import JobLabels
```

Comment:
Do we really need to import v3 here?

Reply:
https://github.com/brighter-ai/redact-client/pull/96/files/1c6774c142a73de4a7cf256820d21ba9d605c914#diff-56b9ba105de1cccc38fa64cf3f185c0ddfcab608c78056dad70ccbc292d4ed3dR31

Reply:
removed it